src/Sylius/Bundle/AdminBundle/Resources/views/Product/Show/_variants.html.twig

```diff
@@ -123,17 +123,17 @@
                                                     </tr>
                                                     {% set currencies = sylius_channels_currencies() %}
                                                     {% for channelPricing in variant.channelPricings %}
+                                                        {% set channel = channelPricing.channelCode|sylius_channel %}
```

Comment:
Same here - we can also think about merging both this template and the pricing one above.